tools/a-star/src/Main.java

```diff
@@ -0,0 +1,95 @@
+import astar.Spot;
+import drawing.DrawingBoard;
+import drawing.MyDrawable;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import static astar.A_Star.A_Star;
+import static astar.A_Star.linkNeighbors;
+import static drawing.DrawingBoard.createGui;
+import static pictureMapper.GetPixelColor.getBoolArr;
+import static pictureMapper.GetPixelColor.getRGBarray;
+
+
+public class Main {
+
+    static final int w = 2;
+    static final int h = 2;
+
+    public static DrawingBoard gui;
```

Comment:
Here is the thing, if you need to have static variables, then you should start creating classes, instead of doing things statically. The way you are doing it is making your code less re-usable

Reply:
These variables we're used just to visualize it with with JPanel and Swing and have nothing to do with the end product, so i didn't bother because its not supposed to be reusable